akka-projection-kafka/src/main/scala/akka/projection/kafka/KafkaOffsets.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.kafka
+
+import java.lang.{ Long => JLong }
+import akka.projection.MergeableOffset
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.apache.kafka.common.TopicPartition
+
+object KafkaOffsets {
```

Comment:
Renamed the GroupedOffsets to KafkaOffsets. This is now a utility class to help us produce valid MergeableOffset instances for Kafka